Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonButton.cs

```diff
@@ -693,33 +693,27 @@ protected override void OnPaint(PaintEventArgs? e)
                 {
                     Text = KryptonManager.Strings.GeneralStrings.Abort;
                 }
-
-                if (DialogResult == DialogResult.Cancel)
+                else if (DialogResult == DialogResult.Cancel)
```

Comment:
This is horrible=..
Please make it a `switch`

Reply:
I actually tried those:
- A normal switch is 3 lines longer?
- A switch expression lacks a default value, which generates a warning?

Which one would you like?